src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -644,17 +645,16 @@ private static Stream<String> streamTextParts(DetailNode node) {
 
     /**
      * Finds the end of a sentence. If a sentence ending period was found, returns the whole string
-     * up to and excluding that period. The end of sentence detection here could be replaced in the
+     * up to and including that period. The end of sentence detection here could be replaced in the
      * future by Java's built-in BreakIterator class.
      *
      * @param text The string to search.
      * @param period The period character to find.
-     * @return The string up to and excluding the period, or null if no ending was found.
+     * @return The string up to and including the period, or empty if no ending was found.
      */
-    @Nullable
-    private static String findSentenceEndingOrNull(String text, String period) {
+    private static String findSentenceEnding(String text, String period) {
```

Comment:
Truthfully, we are not finding the ending of the sentence. We are returning the whole sentence.

Reply:
I don' t understand what you suggest to change here. Could you be more specific?

Reply:
By name alone, finding the end of the sentence suggests to me we return the position of the end. In this case, we are returning the whole "sentence" if a proper period is found. The rest of the javadoc goes along with this at `. If a sentence ending period was found, returns the whole string ...`

It may not be easy to name this. To me a method name would be something like `getSentenceToPeriod` and update the javadoc to follow.

`empty if no ending was found.` => `empty if no sentence with a period was found.`

Reply:
done

Reply:
@rnveach I applied your suggestion. Could you resolve this converstion and unblock the PR?

Reply:
I am not assigned, besides there is discussions in admin discord around the handling of the original contributor's pull request and concerns about process mismanagement and its impact on contributor morale.

Reply:
@rnveach could you explain please why you keep this PR re-assigning and still keeping it blocked? 
<img width="290" alt="image" src="https://github.com/user-attachments/assets/ee373fb5-6cef-4624-92dd-72bef4bbfc18" />

Considering I addressed your notes above, could you clearly explain what blocks this PR and why?

Reply:
DIscussion in discord is still going on. You were added to the discord channel.

Plus I am not assigned, so there is no block.